lib/surface_site_web/components/button.ex

```diff
@@ -7,7 +7,7 @@ defmodule SurfaceSiteWeb.Components.Button do
   def render(assigns) do
     ~H"""
     <button class="button {{ @kind }}" :on-click={{ @click }} style="margin: 0px 5px">
-      <slot/>
+      <slot />
```

Comment:
SurfaceFormatter always ensures a space before the closing `/` on self-closing elements.